docs/changelog.md

```diff
@@ -0,0 +1,88 @@
+---
+id: Changelog
+toc_min_heading_level: 2
+toc_max_heading_level: 2
+---
+
+# Changelog
+
+## October 6, 2023
+
+### Breaking Changes (Code Incompatibility)
+
+❗[responsive-pub/134](https://github.com/responsivedev/responsive-pub/pull/134) 
+and [responsive-pub/135](https://github.com/responsivedev/responsive-pub/pull/135)
+changed the way a new instance of `ResponsiveKafkaStreams` is created. These
+changes apply to `dev.responsive.kafka-client` versions `0.12.0` and above.
+
+Please modify your code to use the constructor place of the static factory 
+method:
+
+```diff showLineNumbers
+-  KafkaStreams streams = ResponsiveKafkaStreams.create(...);
++  KafkaStreams streams = new ResponsiveKafkaStreams(...);
+```
+
+It also removed the timestamped verisons of `StoreSuppliers` since all Responsive
+stores are timestamped. You can apply all of these changes with the following
+`sed` commands:
+
+```bash
+sed -ie "s|ResponsiveKafkaStreams.create|new ResponsiveKafkaStreams|g" ${FILE}
+sed -ie "s|ResponsiveKeyValueParams.timestamped|ResponsiveKeyValueParams.keyValue|g" ${FILE}
+sed -ie "s|ResponsiveKeyValueParams.timestampedFact|ResponsiveKeyValueParams.fact|g" ${FILE}
+sed -ie "s|ResponsiveStores.timestampedKeyValueStore|ResponsiveStores.keyValueStore|g" ${FILE}
+```
+
+❗[responsive-pub/148](https://github.com/responsivedev/responsive-pub/pull/148):
+This change restructured code so to distinguish between public apis and
+internal implementation classes. Some classes, therefore, may have slightly
+shifted package names. If you fail to compile after upgrading versions,
+remove the imports that are not found and re-import them with the same class
+name.
+
+
+### Operator (0.10.0)
+
+✨[responsive-pub/120](https://github.com/responsivedev/responsive-pub/pull/120):
+[Expected Latency](https://docs.responsive.dev/reference/controller-policies#expected-latency)
+diagnoser was introduced that allows you to specify a goal for the expected 
+latency for a record to be processed at a sub-topology (how long it takes
+from the point it is enqueud on to the topic to when it has been processed).
+
+✨[responsive-pub/131](https://github.com/responsivedev/responsive-pub/pull/131):
+Introduced a [Thread Saturation Diagnoser](https://docs.responsive.dev/reference/controller-policies#thread-saturation), 
```

Comment:
Change 'a' -> 'the'? Tbh either "the Foo Diagnoser" or "a Foo Diagnoser" sound ok to me, but it should be consistent with the way we talk about the other diagnoser(s), eg the Expected Latency Diagnoser above. Your choice of a vs the 

```suggestion
Introduced the [Thread Saturation Diagnoser](https://docs.responsive.dev/reference/controller-policies#thread-saturation), 
```